revised/beamswap-proposal.md

```diff
@@ -0,0 +1,330 @@
+
+`How would your development plan be affected if there were no incentives for your project?`
+
+To date, we have not received any incentives from Moonbeam. Despite this, we continue to deliver new products and add functionalities, even providing over $9 million in incentives ourselves to attract liquidity and users to the Moonbeam network. If the network’s support remains insufficient, we may need to consider other parachains and EVM networks as potential new homes for our project and community, where we can collaborate effectively with the network teams.
+
+```
+Can you provide more details on the use cases for connected contracts?
+```
+
+Regarding connected contracts, we plan to incentivize liquidity for the xc-tokens specified in the proposal (DOT, USDT) and integrate Squid’s Axelar-powered router, which facilitates cross-chain swaps and routing. Additionally, we will incorporate Multichain’s recently released v7 router that supports cross-chain swaps. As Multichain is the bridge with the highest Total Value Locked (TVL), it will offer an excellent user experience and reduced slippage. You can find more information here: https://docs.multichain.org/developer-guide/bridge-funds-and-anycall-router-v7.
+
+Furthermore, we are exploring other networks such as Base and Optimism, in line with Moonwell’s approach.
+
+`In case the situation with USDC and Circle deteriorates, the USDC stablecoin would be replaced with USDT (Tether) prior to product launch.`
+
+## 7 Project Overview and Relevant KPIs - brief description of project including historical KPIs on protocol traction and success to date on Moonbeam.
+
+The Beamswap DeFi Hub is among the five leading players in the Moonbeam DeFi arena, according to:
+
+* [Web3Go Moonbeam Analytics](http://app.web3go.xyz/#/MoonbeamPublicDashboard)
+* [DappRadar](https://dappradar.com/rankings/protocol/moonbeam)
+* [Moonbeam Subscan](https://moonbeam.subscan.io/erc20_token)
+
+Beamswap has continuously hit the charts and was listed among/as:
+
+* Top 3 dapps by dapp transactions
+* Top 5 dapps by dapp active addresses
+* Top 3 dapps by number of unique active wallets
+* No. 1 held community token on the chain ($GLINT)
+* No. 1 held secondary token on the chain ($SHARE)
+
+In just over a year since launch, the Beamswap DeFi Hub has accumulated a lifetime volume of USD 450 million, with a peak TVL of USD 100 million, a daily average volume of USD 1 million, and registered over 1.15 million transactions to date.
+
+The Beamswap-native $GLINT token is the most held asset on the Moonbeam Network with 10,900 holders, while the $SHARE token is the most held secondary asset on Moonbeam with 5,000 holders.
+
+![|602x339](upload://fD6fkHq6wKjwisKaZZgPQIGbwJM.jpeg)
```

Comment:
Can you also upload the images to the revised/assets/beamswap folder? Otherwise we'll end up with some empty placeholders.